core/src/main/res/values/strings.xml

```diff
@@ -561,6 +561,10 @@
     <string name="not_auto_downloaded">Not auto downloaded</string>
     <string name="kept_updated">Kept updated</string>
     <string name="not_kept_updated">Not kept updated</string>
+    <string name="pref_feed_settings_dialog_msg">This setting is specific to each podcast. It can be accessed by opening the podcast.</string>
```

Comment:
```suggestion
    <string name="pref_feed_settings_dialog_msg">This setting is specific to each podcast. It can be accessed by opening the podcast and pressing the settings icon.</string>
```